backup.go

```diff
@@ -178,7 +211,8 @@ func backup(ID string) error {
 		return err
 	}
 
-	for _, m := range conf.Mounts {
+    paths = []string{}
```

Comment:
Please make sure your sources are always `gofmt`'d before pushing! It's the best thing since sliced bread, really :smiley:

Reply:
Ah, I didn't know about that command - I'll try it out!